src/main/java/configreader/GetPropertyValues.java

```diff
@@ -8,7 +8,6 @@
 
 public class GetPropertyValues {
     private HashMap<String, String> propsResult = new HashMap<>();
-    private String result = "";
     private FileInputStream inputStream;
 
     public HashMap<String, String> getPropValues() throws IOException {
```

Comment:
Let's not read all properties at a time, but do the method like readProperty(String propertyName) and use it